docs/concepts/getting-started-guide.md

```diff
@@ -3,45 +3,67 @@ id: getting-started-guide
 
 ### Other prerequisites (for Integrations and Automations)
 
-If you want to develop more than just Playbooks and you are looking to create Integrations and Automations, you will also need:
+If you want to develop Integrations and Automations, some coding is required. You will need:
 1. Python (3.7 and above) or Powershell programming experience.
-2. (for Integrations) API or SDK access to your product or solution you want to integrate with.
+1. (for Integrations) API or SDK access to your product or solution you want to integrate with.
 
 ### Technology Partners
 
-If you are a Technology Partner, make sure that you also:
+If you are or want to become a Technology Partner, make sure that you also:
 
 1. Read the [Become a Technology Partner](../partners/become-a-tech-partner) page and follow the steps to sign up and sign the agreements.
-2. Work with the [Cortex XSOAR Alliances Team](mailto:soar.alliances@paloaltonetworks.com) to make sure your use cases have been validated.
+1. Work with the [Cortex XSOAR Alliances Team](mailto:soar.alliances@paloaltonetworks.com) to make sure your use cases have been validated.
+
+## Are you planning to contribute?
+
+This site describes how to create content meant to be published on the XSOAR Marketplace and used by several customers. You must following proper design, development and documentation guidelines to make sure that the content can used in production in large SOCs. While it's always recommended to follow all these guidelines,  some steps, such as the dev environment setup, are mandatory only if you are planning to contribute your content.
+
+The following summarizes some common developer profiles and recommendations on what tools to use to develop and contribute. If you are not creating Integrations and Automations, no coding is required and you can do most of the work just through the XSOAR UI.
+
+|Who|Contributing?|Integrations/Scripts?|Write Code With|Contribute With|
+|--:|------------:|--------------------:|-----------------|---------------|
+|End customer|No|No|N/A|N/A|
+|End customer|No|Yes|[XSOAR IDE](../concepts/xsoar-ide)|N/A|
+|End customer<br/>Individual contributor|Yes|No|N/A|[Cortex XSOAR UI](../contributing/marketplace)|
+|End customer<br/>Individual contributor|Yes|Yes|demisto-sdk + IDE (Pycharm, VSCode, etc.)|GitHub|
+|Technology Partner<br/>Palo Alto Networks Employee|Yes|Either way|demisto-sdk + IDE (Pycharm, VSCode, etc.)|GitHub|
+
+:::note Important Note
+In general, even if you are not contributing, you are never wrong if you develop following the detailed guidelines below, including setting up the development environment and designing and documenting your content properly.
+:::
+
+If, based on the assumptions above,  you are contributing via GitHub and developing with `demisto-sdk` and an IDE, keep reading through the next sectiobn.
```

Comment:
(Forgot to add this)
Let's go over this I am not sure what we are offering here I think most of this is already located in the dev setup. If not let's think about where we should put the missing data.

Reply:
done?